.github/workflows/config-versions-lint.yml

```
@@ -0,0 +1,16 @@
+name: configs-versions-lint
+on: [push, pull_request]
+env:
+  GO_VERSION: 1.23.x
```

```suggestion
  GO_VERSION: 1.24.x
```

Just noticed this so we're consistent with all the other Go version CI usage 👀

---

Fixed https://github.com/gbjk/gocryptotrader/commit/e4fad61e13

Soz, failed to push the fixup commit in isolation. But it's fixed.